omod/pom.xml

```diff
@@ -212,32 +191,14 @@
                                     <version>${uicommonsVersion}</version>
                                     <type>zip</type>
                                     <overWrite>true</overWrite>
-                                    <outputDirectory>src/main/compass/sass-external</outputDirectory>
```

Comment:
Do we need to add "src/main/webapp/resources/styles/lib" to the gitignore as well?

Reply:
Oh, never mind, looks like you added it in the other gitignore, I missed it.